### PR TITLE
Emilybecca

### PR DESCRIPTION
**Pull request by rlgeren**

### CODE REVIEW

**Comment by kbalthaz12** [comment]
Too many people adding branches it broke your branch :(

**Comment by rlgeren** [comment]
@GabrielCoding98 we added you as a reviewer! Please review! THANKS

File: Hey.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 How are you doing?
+Not bad
```
Comment (nahodges5) [inline Hey.txt:2]: Totes!

**Review by nahodges5 — APPROVED** [review]

**Review by grett28 — APPROVED** [review]
Looks great!

**Pull request merged**